backend/src/main/java/ca/bc/gov/hlth/mohums/controller/OrganizationsController.java

```diff
@@ -0,0 +1,65 @@
+package ca.bc.gov.hlth.mohums.controller;
+
+import ca.bc.gov.hlth.mohums.webclient.WebClientService;
+import net.minidev.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.*;
+
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Collection;
+import java.util.Map;
+
+
+@RestController
+public class OrganizationsController {
+
+    //todo: write the url, user and password somewhere else
+    @Value("${spring.datasource.url}")
+    private String url;
+    @Value("${spring.datasource.username}")
+    private String user;
+    @Value("${spring.datasource.password}")
+    private String password;
+    @Value("${spring.datasource.database}")
+    private String database;
+    @Value("${spring.datasource.organization_id}")
+    private String organization_id;
+    @Value("${spring.datasource.organization_name}")
+    private String organization_name;
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public OrganizationsController(WebClientService webClientService, @Value("${config.vanity-hostname}") String vanityHostname) {
+    }
```

Comment:
I don't think this constructor is used, so you can probably delete it. 

SpringBoot dependency injection _can_ call constructors and inject variables, but the constructor body still needs to set the field values otherwise they would be null. `webClientService` and `vanityHostname` aren't fields in this class.s

Reply:
deleted constructor